pkg/api/types.go

```diff
@@ -2793,6 +2793,11 @@ type ExportOptions struct {
 	Export bool `json:"export"`
 	// Should the export be exact.  Exact export maintains cluster-specific fields like 'Namespace'
 	Exact bool `json:"exact"`
+	// If specified:
+	// - if unset, then the result result is returned from remote storage based on quorum-read flag;
+	// - if it's 0, then we simply return what we currently have in cache, no guarantee;
+	// - if set to non zero, then the result is as fresh as given rv.
+	ResourceVersion string
```

Comment:
The pointer to where we parse it:
https://github.com/kubernetes/kubernetes/blob/master/pkg/apiserver/resthandler.go#L137

But I guess the rest question is (mostly about renaming) is still to @smarterclayton